### ISSUE TITLE
[WANTED] Ordnerstruktur für die Ligen am linken Rand

### ISSUE THREAD

**Issue by Carl2620** [floorballdeutschland/saisonmanager-feedback#17]
### Motivation

Der Saisonmanager für Floorball Deutschland hat an die 30 "Ligen" - das ist zu unübersichtlich für eine einzige Liste.

### Vorgeschlagene Lösung

Die Ligen sollten geclustert werden (können), z.B. in "1. FBL Herren", "2. FBL Herren", "Damen", "Jugend", "Meisterschaften" die man dann aufklappen kann um zu den einzelnen Seiten zu kommen.

### Alternativen

-
### Zusätzlicher Kontext

-
### Links zu relevanten Seiten

https://saisonmanager.de/fvd/1204-1-fbl-herren

Wichtig: Nenne keine Personenbezogene oder sensible Daten. Diese sind nach anlegen des Issues an it@floorball.de zu senden

**Comment by signpainter** [comment]
Da könnte man sich Anleihen an fussball.de oder basketball-bund.net nehmen. Dort kann ist das recht gut strukturiert. Das einzige nervige ist, dass man dort, wenn man mal in die jeweilige Liga abgestiegen ist, die Liga schwer wechseln kann.
Interessant wäre auch eine Möglichkeit nach einer Mannschaft/Verein zu suchen und sich bspw. alle Teams eines Vereins anzeigen zu lassen und mit einem Klick in die jeweilige Liga zu wechseln.

**Comment by janhoffmann** [comment]
In diesem Kontext:
Kennt ihr die Favoriten-Funktion?

**Comment by signpainter** [comment]
Ja, die ist hilfreich, wenn man mal gefunden hat, was man sucht.

Für den ersten Zugang zu einer Liga ist's teils etwas unübersichtlich, wenn dann die Endrunden, regionalen Meisterschaften etc dazu kommen.
Da wäre eine Sortierung oder Filterung nach Klasse, Altersklasse Spielform und/oder Geschlecht hilfreich.

**Comment by AdrianMuehle** [comment]
Die Faviroten-Funktion ist sehr hilfreich, sollte aber in dem Kontext nach Oben gezogen werden. Es macht kein Sinn durch alle LIgen zu scrollen, um am Ende seine Favoriten zu sehen. Ich möchte gern erst die Favoriten verwalten und dann die Ligen durchscrollen. Könnt ihr das realisieren?

**Comment by dkehne** [comment]
Favoriten stehen inzwischen oben in der Navigation. Eine echte Gruppierung/Ordnerstruktur der Ligen gibt es noch nicht.

**Comment by dkehne** [comment]
@AdrianMuehle würde mich hier nochmal über deine aktuelle Einschätzung freuen. Favoriten gibts nun auch auf Teamebene...
